xtask/Cargo.toml

```diff
@@ -17,3 +17,4 @@ helix-core = { path = "../helix-core" }
 helix-view = { path = "../helix-view" }
 helix-loader = { path = "../helix-loader" }
 toml = "0.8"
+itertools = "0.12.1"
```

Comment:
we don't want to pull in extra dependencies, particularly not itertools

Reply:
ok so use what's available to remove the Vector Duplication string?

Reply:
I'm not on my computer right now, but if that's a vector then it already has `dedup` methods

Reply:
You could possibly collect to a `HashMap` or a `IndexMap` (if order is important)?

Reply:
OK redid it without itertools. let me know what you think.